app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,41 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <style>
-      /* Email styles need to be inline */
+      * {
```

Comment:
Consider using the `body` selector here instead of `*`.

`*` won't work in as many email clients, according to this page: https://templates.mailchimp.com/resources/email-client-css-support/